### DISCUSSION TITLE
Disable rainbow yaml tags

### DISCUSSION THREAD

**Discussion by ghost** [izhangzhihao/intellij-rainbow-brackets#2639] [Q&A] [answered]
Is there a way to disable it for a specific language in rainbow-brackets? The yaml added this time is rather less readable, so I want to disable it only in the yaml.

**Comment by izhangzhihao** [comment]
Try this for now:

Edit the config file at https://github.com/izhangzhihao/intellij-rainbow-brackets#config-file-path

set `rainbowifyTagNameInYAML` to false.

I will add this to a new option in the next release.

**Comment by izhangzhihao** [comment, marked as answer]
<img width="984" alt="image" src="https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/12044174/e10f0adc-0ee9-490e-ae25-f5db92ad2b5f">
